soc/arm/intel_socfpga_std/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+# Description:
+# Adding support for Intel Cyclone V SoC
+
+add_subdirectory(${SOC_SERIES})
```

Comment:
Please add newlines at the end of all files. All editors have an option to do that.